images/atomic-openshift-descheduler.yml

```diff
@@ -1,10 +1,10 @@
 container_yaml:
   go:
     modules:
-    - module: github.com/kubernetes-incubator/descheduler
+    - module: sigs.k8s.io/descheduler
```

Comment:
Wondering if this line has any effect since the latest descheduler image was built successfully [1] (private link). Though, needed for consistency.

[1] https://brewweb.engineering.redhat.com/brew/buildinfo?buildID=1082885